nservicebus/messaging/send-a-message.md

```diff
@@ -29,7 +29,7 @@ partial:batch-sends-performance
 
 ## Inside the incoming message processing pipeline
 
-In some cases, messages might need to be sent as part of handling a message such as, inside a regular message handler, a saga, or in some advanced cases as part of extending the message handling pipeline. In this scenario, all the send operations are linked to the incoming message. The send operations take part in the same transaction scope as that of the message handler, thereby ensuring that the send operations rollback if the handling of the message fails at any stage of the message processing pipeline.
+Messages often need to be sent as part of handling an incoming message. When running in a [transaction mode](/nservicebus/transports/transactions.md) that supports it, these send operations take part in the same transaction as that of the message handler, thereby ensuring that the send operations roll back if the handling of the message fails at any stage of the message processing pipeline.
```

Comment:
@timbussmann - doesn't the following say that?

>thereby ensuring that the send operations roll back if the handling of the message fails at any stage of the message processing pipeline.

How do you think we should clarify it?